src/test/resources/integration/updateUsertaxVerificationStatus-response.json

```diff
@@ -0,0 +1,33 @@
+{
+  "token": "usr-b8e7ff1d-a3c6-45a0-ae0a-62b74580caca",
```

Comment:
A `t` in the file name is lower case when it should be upper case.

Reply:
done